var/spack/repos/builtin/packages/py-setuptools-git/package.py

```diff
@@ -0,0 +1,36 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PySetuptoolsGit(PythonPackage):
+    """Setuptools revision control system plugin for Git"""
+
+    homepage = "https://pypi.python.org/pypi/setuptools-git"
+    url      = "https://pypi.io/packages/source/s/setuptools-git/setuptools-git-1.2.tar.gz"
+
+    version('1.2', '40b2ef7687a384ea144503c2e5bc67e2')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
Looking at this:

https://github.com/pyfidelity/setuptools-git-version/blob/f80104e1cc67f8801b27c1a99071315e24c88aee/setuptools_git_version.py#L5

It seems that the package also depends on `git`.

Reply:
Thanks for the review @alalazo, added the git dependency.